.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    open-pull-requests-limit: 999
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    open-pull-requests-limit: 999
+    ignore:
+      - dependency-name: '*'
+        update-types: ['version-update:semver-major']
```

Comment:
i'd rather get notified of major updates, it may mean that this module needs to be updated for any breaking changes. but i'd rather get notified and have that option, than not know